src/components/calcite-color/calcite-color.e2e.ts

```diff
@@ -232,14 +232,12 @@ describe("calcite-color", () => {
     it("value as property", async () => {
       // initialize page with calcite-color to make it available in the evaluate callback below
       const page = await newE2EPage({
-        html: "<calcite-color></calcite-color>"
+        html: `<calcite-color id="test"></calcite-color>`
       });
-      await page.setContent("");
 
       await page.evaluate(async (color) => {
-        const picker = document.createElement<HTMLCalciteColorElement>("calcite-color");
+        const picker = document.getElementById("test") as HTMLCalciteColorElement;
```

Comment:
This breaks the test setup actually. This is meant to test programmatic creation of the component and assigning the value. I think my comment may need more detail, but this weird setup (create and clear a calcite component) is needed to bootstrap components when creating them in the browser context.

If this is to fix the following type error:

```
TS2344: Type 'HTMLCalciteColorElement' does not satisfy the constraint '"listing" | "xmp"'.   Type 'HTMLCalciteColorElement' is not assignable to type '"xmp"'.
```

Can you try 

```tsx
const picker = document.createElement("calcite-color") as HTMLCalciteColorElement;
```

instead?

Thanks for fixing, btw!

Reply:
Got it. The weird part of this test is that the page is being created with a calcite-color but then removed. Can we get rid of that? @jcfranco

Reply:
No, this is the bootstrap part I mentioned. If you take that away, the color component in doesn't get upgraded in the browser context.